COMETwebapp/ViewModels/Pages/TabsViewModel.cs

```diff
@@ -84,7 +86,11 @@ public TabsViewModel(ISessionService sessionService, IServiceProvider servicePro
         public TabbedApplication SelectedApplication
         {
             get => this.selectedApplication;
-            set => this.RaiseAndSetIfChanged(ref this.selectedApplication, value);
```

Comment:
Why this change?

Reply:
Because if I select an application that is already selected, the method InitializeViewModelBasedOnApplication won't be triggered. This is an issue atm, because the opentab component sets the current application for the tab, and if the application selected in the form is already selected, nothing happens inside the tabvm. This flow will likely to be changed when we implement support for multiple models in this page.